fastchat/serve/openai_api_server.py

```diff
@@ -15,6 +15,7 @@
 import logging
 
 import os
+import tiktoken
```

Comment:
1. Move this to the third party section
2. Add the new dependency to https://github.com/lm-sys/FastChat/blob/32119fa743f06f1908ff1ec99f58c6455158b735/pyproject.toml#L18.